grid2op/Episode/EpisodeData.py

```diff
@@ -223,10 +223,14 @@ def set_meta(self, env, time_step, cum_reward, seed):
             self.meta["env_type"] = "{}".format(type(env).__name__)
             self.meta["nb_timestep_played"] = time_step
             self.meta["cumulative_reward"] = cum_reward
-            if seed is None:
-                self.meta["env_seed"] = seed
+            if env_seed is None:
```

Comment:
`is None` => `is not None`

Reply:
Why ? I think it's correct, if it's a `None` then you let it be. Otherwise you convert it to int, no ?